app/ide-desktop/lib/client/src/file-associations.ts

```diff
@@ -146,6 +148,29 @@ export function handleOpenFile(openedFile: string): string {
         }
         logger.error(e)
         electron.dialog.showErrorBox(common.PRODUCT_NAME, message)
+        // eslint-disable-next-line no-restricted-syntax
```

Comment:
@somebody1234 why we still need such strange linter config in code? I remember we were talking about fixing linter config some time ago and making such annotations not needed anymore. What is the status of this topic? :)

Reply:
the cause of this specific one is bad eslint config, it should be able to be removed once #6267 is merged